setup.py

```diff
@@ -0,0 +1,128 @@
+        new_version[-1] = str(int(new_version[-1]) + 1)  # Bump the final part
+        self.version = ".".join(new_version)
+
+    def finalize_options(self):
+        pass
+
+    def run(self):
+
+        print('old version: %s  new version: %s' %
+              (metadata['version'], self.version))
+        try:
+            input('Press enter to confirm, or ctrl-c to exit >')
+        except KeyboardInterrupt:
+            raise SystemExit("\nNot proceeding")
+
+        old = "__version__ = '%s'" % metadata['version']
+        new = "__version__ = '%s'" % self.version
+        module_content = read_module_contents()
+        with open('version.py', 'w') as fileh:
+            fileh.write(module_content.replace(old, new))
+
+        old = 'Version: %s' % metadata['version']
+        new = 'Version: %s' % self.version
+        spec_file = read_spec_contents()
+        with open('monitoring-integration.spec', 'w') as fileh:
+            fileh.write(spec_file.replace(old, new))
+
+        # Commit everything with a standard commit message
+        cmd = ['git', 'commit', '-a', '-m', 'version %s' % self.version]
+        print(' '.join(cmd))
+        subprocess.check_call(cmd)
+
+
+class ReleaseCommand(Command):
+    """Tag and push a new release."""
+
+    user_options = [('sign', 's', 'GPG-sign the Git tag and release files')]
+
+    def initialize_options(self):
+        self.sign = False
+
+    def finalize_options(self):
+        pass
+
+    def run(self):
+        # Create Git tag
+        tag_name = 'v%s' % version
+        cmd = ['git', 'tag', '-a', tag_name, '-m', 'version %s' % version]
+        if self.sign:
+            cmd.append('-s')
+        print(' '.join(cmd))
+        subprocess.check_call(cmd)
+
+        # Push Git tag to origin remote
+        cmd = ['git', 'push', 'origin', tag_name]
+        print(' '.join(cmd))
+        subprocess.check_call(cmd)
+
+        # Push package to pypi
+        # cmd = ['python', 'setup.py', 'sdist', 'upload']
+        # if self.sign:
+        #    cmd.append('--sign')
+        # print(' '.join(cmd))
+        # subprocess.check_call(cmd)
+setup(
+    name="monitoring-integration",
+    version="1.0.0",
+    author="Rishubh Jain",
+    author_email="risjain@redhat.com",
+    description=("Integration of tendrl with grafana and"
+                 " creating default dashboard in grafana"),
+    license="LGPL-2.1+",
+    keywords="",
+    packages=find_packages(exclude=["*.tests", "*.tests.*", "tests.*",
+                                    "tests"]),
+    url="http://www.redhat.com",
+    namespace_packages=['tendrl'],
+    long_description="",
+    classifiers=[
+        "Development Status :: 4 - Beta"
+    ],
+    zip_safe=False,
+    install_requires=[
+        "ruamel.yaml",
+        "maps",
+        "requests"
+        ],
+    entry_points={
+        'console_scripts': [
+            'monitoring-integration = tendrl.monitoring-integration:main'
```

Comment:
* tendrl.monitoring_integration
Is this tendrl.monitoring_integration hierarchy is present?